adventurer/templates/home.html

```diff
@@ -0,0 +1,37 @@
+<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"default"}} -->
+<div class="wp-block-group"><!-- wp:cover {"url":"https://adventurertheme.mystagingwebsite.com/wp-content/uploads/2022/11/costa_rica_coconuts.png","id":160,"dimRatio":50,"minHeight":90,"minHeightUnit":"vh","style":{"spacing":{"padding":{"right":"0","left":"0","top":"0","bottom":"0"}}}} -->
+<div class="wp-block-cover" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0;min-height:90vh"><span aria-hidden="true" class="wp-block-cover__background has-background-dim"></span><img class="wp-block-cover__image-background wp-image-160" alt="" src="https://adventurertheme.mystagingwebsite.com/wp-content/uploads/2022/11/costa_rica_coconuts.png" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:template-part {"slug":"header","tagName":"header"} /-->
```

Comment:
Should we change the URL here to be local?

Reply:
Yeah. Working on this theme now!